utils/rpcd-mod-attendedsysupgrade/files/attendedsysupgrade.acl

```diff
@@ -0,0 +1,26 @@
+{
+	"attendedsysupgrade": {
+		"description": "attended sysupgrade via rpcd",
+		"read": {
+			"ubus": {
+				"attendedsysupgrade": [
+					"sysupgrade"
+				], 
+				"system": [
+					"board"
+				],
+				"uci": [ 
+					"get" 
+				]
+			},
+			"uci": [ 
+				"*" 
+			]
+		},
+		"write": {
+			"cgi-io": [
```

Comment:
The package itself does not use "cgi-io", I suppose this is required by some ui not included here. I would be better to bundle the "cgi-io" ACL with the gui requiring it.

Reply:
The other functions aren't used neither directly by the package, should I move them all to luci-app-attendedsysupgrade?